frontend/src/service/api-service.tsx

```diff
@@ -45,6 +45,11 @@ export const createNewMilestone = (token: string, milestoneDto: MilestoneDto) =>
     .post('/api/project-planner/milestone', milestoneDto, getAuthHeaders(token))
     .then(response => response.data)
 
+export const upDateMilestone = (token: string, milestoneDto: MilestoneDto) =>
```

Comment:
```suggestion
export const updateMilestone = (token: string, milestoneDto: MilestoneDto) =>
```
This is a weird way of writing update :)